readme.md

```diff
@@ -40,7 +40,18 @@ importance (although feel free to ask them to try to learn more about _why_
 12. Note, in order to be able to perform these responsibilities it's important
 that POs have been given the authority and means necessary to complete them. :muscle:
 
-* product backlog: a prioritised features list for the product found on github.
+*product backlog: a prioritised features list for the product found on github.
+
+## Product Owners should create issues rather than dwylers.
+
+### Why?
+
```

Comment:
The points in this section make sense but I'm feeling that it's a change in 'voice'. Whereas most of this repo is written as if speaking to POs, these bullet points feel like they're written _about_ POs but _not_ as if they were the ones reading this.